Automattic/jetpack
# Generate query string for the JITM request

Fixes https://github.com/Automattic/dotcom-forge/issues/2891

## Proposed changes:

* generate a query string for the `query_string` parameter

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this version on a WoA site
* Go to `/plugins`
* Search for a plugin where JITMs should be shown, eg. `woocommerce`, `apple pay`, or `elementor`
* Check that JITMs are shown on top of the search results
